customer-commons/src/main/java/no/unit/nva/customer/model/CustomerDao.java

```diff
@@ -43,8 +43,11 @@ public class CustomerDao implements Typed {
     public static final String CRISTIN_ID = "cristinId";
     public static final String TYPE = "Customer";
     public static final Set<VocabularyDao> EMPTY_VALUE_ACCEPTABLE_BY_DYNAMO = null;
+    public static final Set<PublicationInstanceTypes>
+        PUBLICATION_INSTANCE_TYPES_EMPTY_VALUE_ACCEPTABLE_BY_DYNAMO = null;
     public static final TableSchema<CustomerDao> TABLE_SCHEMA = TableSchema.fromClass(CustomerDao.class);
     public static final String VOCABULARIES_FIELD = "vocabularies";
+    public static final String PUBLICATION_TYPES_FIELD = "allowFileUploadFor";
```

Comment:
Name of constant should relate to field name. ALLOW_FILE_UPLOAD_FOR_FIELD is the appropriate name.